src/deluge/model/drum/non_audio_drum.cpp

```diff
@@ -57,7 +57,7 @@ TimelineCounter* playPositionCounter) { if (whichModEncoder == 0) { channelEncod
 int8_t NonAudioDrum::modEncoderAction(ModelStackWithThreeMainThings* modelStack, int8_t offset,
                                       uint8_t whichModEncoder) {
 
-	if ((getCurrentUI() == &instrumentClipView || getCurrentUI() == &automationClipView)
+	if ((getCurrentUI() == &instrumentClipView || getCurrentUI() == &automationView)
```

Comment:
same here (would just be rootUI == instrumentClipView if you use the UI stack)

Reply:
> same here (would just be rootUI == instrumentClipView if you use the UI stack)

See explanation above. I don't think it's an issue. This is to allow you to use mod encoders for kit rows while in automation overview.